fixed.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1beta2
```

Comment:
Was this file added intentionally?

Reply:
noooo! sorry! I made a mistake when going back and forth! I should have reviewed this more carefully before asking for feedback!

Reply:
I was supposed to have deleted the yaml and deletes the fix.go`` by accident instead!